packages/playwright-core/src/utils/index.ts

```diff
@@ -191,12 +191,10 @@ export function constructURLBasedOnBaseURL(baseURL: string | undefined, givenURL
 
 export function wrapInASCIIBox(text: string, padding = 0): string {
   const lines = text.split('\n');
-  const maxLength = Math.max(...lines.map(line => line.length));
-  return [
-    '╔' + '═'.repeat(maxLength + padding * 2) + '╗',
-    ...lines.map(line => '║' + ' '.repeat(padding) + line + ' '.repeat(maxLength - line.length + padding) + '║'),
-    '╚' + '═'.repeat(maxLength + padding * 2) + '╝',
-  ].join('\n');
+  const maxLineLength = Math.max(...lines.map(line => line.length));
+  const separatorLength = process.stdout.isTTY ? process.stdout.columns : maxLineLength;
```

Comment:
I think `columns` could be zero in some cases, perhaps the following would work better?

```suggestion
  const separatorLength = process.stdout.columns || maxLineLength;
```